src/aggregate.js

```diff
@@ -56,6 +55,7 @@ export default async function aggregate(calls, config) {
   }, {});
 
   calls = calls.map(({ call, target, returns }) => {
+    if (!target) target = config.multicallAddress;
```

Comment:
what case is this for?

Reply:
The new `ABIEncoderV2` approach allows us to call into the same contract, so the helper functions can put pulled into `Multicall.sol`.

This being the case, a way to easily call these helper functions could be to specify the function name, e.g. `getEthBalance(address)(uint256)` but omit the target, which would then default to the multicall contract address (where the helper functions would now be).

Reply:
The new `ABIEncoderV2` approach allows us to call into the same contract, so the helper functions can be pulled into `Multicall.sol`.

This being the case, one way to easily call these helper functions would be to specify the function name, e.g. `getEthBalance(address)(uint256)` but omit the target, which would then default to the multicall contract address (where the helper functions would now be).